ads/google/a4a/utils.js

```diff
@@ -414,13 +420,19 @@ export function extractAmpAnalyticsConfig(a4a, responseHeaders) {
     const slotId = a4a.element.getAttribute('data-amp-slot-index');
     const qqid = (responseHeaders && responseHeaders.has(QQID_HEADER))
         ? responseHeaders.get(QQID_HEADER) : 'null';
-    config['requests']['visibilityCsi'] =
-        'https://csi.gstatic.com/csi?fromAnalytics=1' +
-        `&c=${correlator}&slotId=${slotId}&qqid.0=${qqid}`;
+    opt_initTime = opt_initTime || 0;
+    opt_deltaTime = opt_deltaTime || 0;
```

Comment:
Could you make these an out-of-band value so that we can filter them at analysis time, if necessary?  Should still be a number, so that they will flow through consumer pipelines okay, but recognizably wrong.  `-1` perhaps?